### ISSUE TITLE
Migrate Concourse jobs from iaas-workers pool to default workers (After #1465 completes)

### ISSUE THREAD

**Issue by apburnes** [cloud-gov/product#1466]
In order to support a least privilege model, we want to move all jobs running on the Concourse **iaas-workers** to be migrated to the default workers and have there credentials passed to individual pipelines and be tailored to the pipeline's specific use case. Dependent on the completion of #1465. Based on the findings from #1456 and the next steps meeting notes

## Acceptance Criteria
* [ ] All jobs will be migrated to the default Concourse **workers** and the **iaas-workers** pool will be removed
* [ ] Pipelines function properly by passing the credentials to the pipeline directly
* [ ] A new issue is created for each of the pipelines to be migrated ([see list](https://docs.google.com/document/d/1yO5tDEeJ2jIGvmF0lyTu28WOnAZUyPw3GAjawrDtGL4/edit#))

--- 

## Security considerations
Improves our least privilege model for access and permissions to resources and infrastructure

## Implementation sketch
[links to background notes, sketches, and/or relevant documentation
* [ ] Select a pipeline from the [list](https://docs.google.com/document/d/1yO5tDEeJ2jIGvmF0lyTu28WOnAZUyPw3GAjawrDtGL4/edit#)
* [ ] Create an issue outlining the migration work
* [ ] Add proper credentials to the pipeline for job to consume
* [ ] Remove relevant pipeline jobs from the **iaas-workers** pool
